Project.toml

```
@@ -10,6 +10,6 @@ Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 Sockets = "6462fe0b-24de-5631-8697-dd941f90decc"
 
 [compat]
-julia="^1.5"
-HTTP = "^0.9"
+julia="^1.7"
+HTTP = "1"
```

Given that you didn't change anything else in this PR, i think it should be this:
```suggestion
HTTP = "0.9, 1"
```


There's no reason for this package to _force_ 1.0 if some other package can't upgrade yet. This change would mean that a downstream application would be stuck on an older version of this package if they can't upgrade HTTP yet, and i don't think that's actually required, yeah?

---

On the other hand, i guess it doesn't hurt to be explicit about requirements, so lower-bounding at 1.0 from now on seems reasonable, since it's a major release. so maybe ignore this